packages/aft/pubspec.yaml

```diff
@@ -9,14 +9,14 @@ environment:
 dependencies: 
   args: ^2.3.0
   async: ^2.9.0
-  aws_common: ^0.2.0
+  aws_common: ">=0.2.3 <0.3.0"
   checked_yaml: ^2.0.0
   cli_util: ^0.3.5
   collection: ^1.16.0
   flutter_tools:
     git:
       url: https://github.com/flutter/flutter.git
-      ref: f3d182f4b4e3c5e5cde322f5a14e49e9a50cc083
+      ref: 0df0e2ea1f7fe5021c3593d61903e70bdbda8107
```

Comment:
Curious how do we determine what `ref` to use?

Reply:
Trial and error unfortunately. It's finding the latest versions that don't have conflicting dependencies + a bit of tweaking dependency_overrides to get it all to align.